src/Ilios/CoreBundle/Entity/Cohort.php

```diff
@@ -130,7 +131,7 @@ public function setProgramYear(ProgramYearInterface $programYear = null)
      */
     public function getProgramYear()
     {
-        return $this->programYear;
+        return $this->programYear && !$this->programYear->isDeleted()?$this->programYear:null;
```

Comment:
this will return a boolean value. not null or the program year object.

http://viper-7.com/ee5l8Z